src/components/Layout/demo/ColPresenter/ColPresenter.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {Box} from '../Box/Box';
+import {Col, ColProps} from '../../Col/Col';
+
+// @ts-ignore-error
```

Comment:
What do this directive?

Reply:
ignore type error in demo component. 
I omit nullable values here